### PR TITLE
workflows: ensure there is a dist build available to consume when creating a tag for the release

### PR DESCRIPTION
**Pull request by KKoukiou**
I have not tested it - I will do it tomorrow.

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -1,5 +1,8 @@
 name: release
 on:
+  workflow_run:
+    workflows: publish-dist
+    types: [completed]
   push:
     tags:
```
Comment (martinpitt) [inline .github/workflows/release.yml:7]: This doesn't quite work -- the workflow will run *both* on pushing the tag, *and* for *every* PR opening and landing.

Rather, keep the `push: tags` trigger and teach the download-dist call in `Makefile` something like `${DOWNLOAD_DIST_OPTIONS:-}` and in this workflow set `DOWNLOAD_DIST_OPTIONS=--wait`.

**Review by martinpitt — CHANGES_REQUESTED** [review]

**Comment by martinpitt** [comment]
FTR, quick hack if this somehow doesn't work: add `GITHUB_BASE=fake/nonexisting` to the release workflow to force the dist download to fail.

File: .github/workflows/release.yml (hunk 1 above)
Comment (KKoukiou) [reply .github/workflows/release.yml]: I tested it locally, outside of the release workflow and it works like a charm

I am not sure how I should actually test it by triggering the real release-workflow, I don't want to publish something accidentally from a run from my fork.

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -27,6 +27,6 @@ jobs:
       - name: Run cockpituous
         run: |
           # override GitHub's bind mount from host, we don't want anything from there and it interferes with ssh
-          export HOME=$(getent passwd $(id -u) | cut -f6 -d:)
+          export HOME=$(getent passwd $(id -u) | cut -f6 -d:) DOWNLOAD_DIST_OPTIONS=--wait
```
Comment (martinpitt) [inline .github/workflows/release.yml:30]: I'd prefer this on a separate line to make it easier to read, but not a blocker.

**Review by martinpitt — APPROVED** [review]
Looks good, thank you!

**Comment by martinpitt** [comment]
@KKoukiou : Hmm, is this complete? I thought you also needed to add that auto-generated license file to `%license`?

**Comment by KKoukiou** [comment]
> @KKoukiou : Hmm, is this complete? I thought you also needed to add that auto-generated license file to `%license`?

That was fixeed in another PR. See

commit dfd041034a52905c62a3bcc4a3e94ef6687cb8a6
Author: Katerina Koukiou <kkoukiou@redhat.com>
Date:   Mon Mar 29 13:49:17 2021 +0200

    Fix License field in the package spec file

**Review by martinpitt — APPROVED** [review]
Of course, sorry for mixing these up. Thanks!

**Pull request merged**